core/internal/cluster/kafka_cluster.go

```diff
@@ -278,3 +299,38 @@ func (module *KafkaCluster) getOffsets(client helpers.SaramaClient) {
 		return false
 	})
 }
+
+func (module *KafkaCluster) reapNonExistingGroups(client helpers.SaramaClient) {
+	kafkaGroups, err := client.ListConsumerGroups()
+	if err != nil {
+		module.Log.Warn("failed to get the list of available consumer groups", zap.Error(err))
+	}
+
+	req := &protocol.StorageRequest{
+		RequestType: protocol.StorageFetchConsumers,
+		Reply:       make(chan interface{}),
```

Comment:
Why no type here? `interface{}`

Reply:
that's the type of `Reply`, we cannot use a concrete type here